### DISCUSSION TITLE
Add order conditions to payment geteways, including shipping method condition

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/commerce#3827] [Ideas] [answered]
Example use case: 

We have two shipping methods - courier delivery and parcel locker delivery.
For courier delivery we want to enable "stripe" and "give money to the courier" payment gateway.
For parcel locker there is no one to give money to, so just "stripe" payment gateway should be enabled.

Could you please consider implementing this? Right now i just define such relationship in template directly using twig template, but this gives website owner no ability to adjust it using control panel.

**Comment by lukeholder** [comment, marked as answer]
See https://github.com/craftcms/commerce/discussions/3828
